src/App.css

```diff
@@ -1,9 +1,67 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&family=Montserrat:wght@300;400;600&family=Poppins:wght@300;400;600;800&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&family=Montserrat:wght@300;400;600&family=Poppins:wght@300;400;600;800&family=Roboto+Slab:wght@300;400;600&display=swap');
+
+*,
+*::after,
+*::before {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Montserrat', sans-serif;
+  background-color: #f5f6fa;
+  padding: 50px;
+}
+
+.main-container {
+  box-shadow: rgba(100, 100, 111, 0.2) 0 7px 29px 0;
+  padding-bottom: 60px;
+}
+
+.header {
+  background-color: white;
+  padding: 30px 0;
+  margin-bottom: 30px;
+}
+
 nav {
   display: flex;
   justify-content: space-between;
   align-items: center;
 }
 
+.navigation {
+  display: flex;
+  align-items: center;
+
+  /* width: 100%; */
+}
+
+.logo {
+  font-size: 1.875rem;
+  font-weight: 700;
+  color: #0290ff;
+}
+
+/* h1,
+h2 {
+  font-weight: lighter;
+} */
```

Comment:
- [ ] Kindly remove unused codes from your codebase. It's a good practice to not have unused codes in your codebase as that will make your code clean and lean and professional. Please remove it so your codebase looks clean and professional 👍